### PR TITLE
CleanProFinder.Server: service provider sign up

### PR DESCRIPTION
**Pull request by s1uns**

### CODE REVIEW

**Review by Wednumi — APPROVED** [review]

**Review by Wednumi — CHANGES_REQUESTED** [review]
You forgot about endpoint in the controller

**Comment by s1uns** [comment]
> You forgot about endpoint in the controller

**Comment by s1uns** [comment]
> You forgot about endpoint in the controller

Fixed

File: CleanProFinder/CleanProFinder.Server/Controllers/AccountController.cs [hunk 1]
```diff
@@ -39,5 +39,14 @@ public AccountController(IMediator mediator, IMapper mapper)
             var result = await _mediator.Send(request, cancellationToken);
             return ConvertFromServiceResponse(result);
         }
+
+        [HttpPost("service-provider/account/create")]
```
Comment (Wednumi) [inline CleanProFinder/CleanProFinder.Server/Controllers/AccountController.cs:43]: please change endpoint to new format account/service-provider/create (initial created by me was wrong) and add docs with example of user signup

**Review by Wednumi — CHANGES_REQUESTED** [review]

File: CleanProFinder/CleanProFinder.Server/Controllers/AccountController.cs (hunk 1 above)
Comment (s1uns) [reply CleanProFinder/CleanProFinder.Server/Controllers/AccountController.cs]: fixed

**Pull request merged**